assets/env.js

```diff
@@ -0,0 +1,7 @@
+(function(window) {
```

Comment:
These files should probably go into the 'assets' directory that we already had under 'src' i think?